lib/alg-gost3411-2012-core.c

```diff
@@ -149,10 +149,13 @@ g(uint512_u *h, const uint512_u *N, const unsigned char *m)
 static inline void
 stage2(GOST34112012Context *CTX, const unsigned char *data)
 {
-    g(&(CTX->h), &(CTX->N), data);
+    union uint512_u m;
```

Comment:
Looks like the word `union` here is unneeded (implied as part of the `typedef`), right? If so, I'm mildly surprised compilers are happy to ignore it.

Reply:
Fixed in 5cce1ebe4e8e303bab593af694c9d30c51cf5eb4.